samples/dnn/models.yml

```diff
@@ -15,8 +15,10 @@ opencv_fd:
   rgb: false
   sample: "object_detection"
 
+# YOLO4 object detection family from Darknet (https://github.com/AlexeyAB/darknet)
 # YOLO object detection family from Darknet (https://pjreddie.com/darknet/yolo/)
-# Might be used for all YOLOv2, TinyYolov2 and YOLOv3
+# Might be used for all YOLOv2, TinyYolov2, YOLOv3, YOLOv4 and TinyYolov4
+
 yolo:
```

Comment:
Keep style consistent in this file and remove added extra line.

Reply:
done